static/css/style.css

```diff
@@ -2,9 +2,9 @@ body, input {
     font-size: 17px;
     line-height: 27px;
     -webkit-font-smoothing: subpixel-antialiased;
-    font-family:-apple-system-font, BlinkMacSystemFont, 'Helvetica Neue', sans-serif;
+    font-family:-apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Ubuntu, roboto, noto, arial, sans-serif;
```

Comment:
I actually like the current font better (at least on mac) - current:
<img width="383" alt="image" src="https://user-images.githubusercontent.com/11841271/160285191-b58ec332-e96f-4282-941b-f495fd25a0b6.png">

new:
<img width="384" alt="image" src="https://user-images.githubusercontent.com/11841271/160285183-7358d09f-968a-41a3-a2ca-c91aa3b16a64.png">

Also, line spacing in the paragraph is a bit too large + font size is too small

Reply:
I liked the colors and the design of the "guess" button + textbox!